bindings/python/pymongoarrow/api.py

```diff
@@ -336,6 +337,10 @@ def write(collection, tabular):
             collection.insert_many(cur_batch)
         except BulkWriteError as bwe:
             raise ArrowWriteError(_transform_bwe(dict(bwe.details), cur_offset)) from bwe
+        except pymongo.errors.PyMongoError as pme:
+            raise ArrowWriteError(
```

Comment:
Could we include `nInserted` and `writeConcernErrors` as well to match the BulkWriteError error case?

Reply:
I'm confused why you're in favor of adding what are essentially null fields here, but in the instance above you are not in favor of keeping null fields that we are going to make use of in the future.

Reply:
I think we should add writeConcernErrors for consistency with the other error case. For `nInserted` it is not null, it's very possible to insert a batch of documents and then run into a non-BulkWriteError on the next batch. So adding `nInserted` is actually required.